people/ondrej-bojar.md

```diff
@@ -0,0 +1,12 @@
+---
+grand_parent: Community
+parent: People
+title: Ondřej Bojar
+description: Machine translation researcher
```

Comment:
Hi Vilém! 

Thank you for the PR!

Other `People` pages (like [Alon Lavie](https://github.com/machinetranslate/machinetranslate.org/blame/master/people/alon-lavie.md) include a wider `description`, but I see inconsistencies among these pages.  Do you think it's best to include more info, ie. `Machine translation researcher and associate professor`, in all these pages?

Reply:
I don't have a strong opinion on this. "Associate/assistant" professors tend to change titles often so we need to keep up with that.